community/examples/xpk-gke-a3-megagpu.yaml

```diff
@@ -70,25 +72,40 @@ deployment_groups:
       name: $(vars.deployment_name)-gp-np-1
       group_placement_max_distance: 2
 
-  - id: a3_megagpu_pool_0
+  - id: system_pool
+    source: modules/compute/gke-node-pool
+    use: [gke_cluster]
+    settings:
+      name: system-pool
+      machine_type: "e2-standard-16"
+      autoscaling_total_min_nodes: 2
+      autoscaling_total_max_nodes: 8
+      initial_node_count: 2
+      zones: [$(vars.zone)]
+      host_maintenance_interval: PERIODIC
+    outputs: [instructions]
+
+  - id: a3_megagpu_pool_np_0
```

Comment:
why do we have two nodepools?

Reply:
The second modepool has no default taints and can handle system software which has not been modified by our team.